definition/http.go

```diff
@@ -20,5 +20,22 @@ type Request struct {
 type Response struct {
 	StatusCode int `json:"statusCode"`
 	headers
-	Body string `json:"body"`
+	Persisted  Persisted `json:"persisted"`
+	Body       string    `json:"body"`
+	BodyAppend string    `json:"BodyAppend"`
```

Comment:
Is this necessary? The **BodyAppend** inside the persisted struct is not enough?